test/microbench/fam-api-mb/fam_microbenchmark.cpp

```diff
@@ -63,28 +69,67 @@ TEST(FamCreateDestroyRegion, FamCreateDestroyRegionMultiple1) {
         sprintf(regionInfo, "%s_%d", testRegionLocal, i);
         EXPECT_NO_THROW(descLocal[i] = my_fam->fam_create_region(
                             regionInfo, BLOCK_SIZE, 0777, RAID1));
-        EXPECT_NE((void *)NULL, descLocal);
-
+        EXPECT_NE((void *)NULL, descLocal[i]);
+    }
+    EXPECT_NO_THROW(my_fam->fam_barrier_all());
+    EXPECT_NO_THROW(rpc->reset_profile());
+    EXPECT_NO_THROW(my_fam->fam_barrier_all());
+    for (int i = 0; i < NUM_MM_ITERATIONS; i++) {
         EXPECT_NO_THROW(my_fam->fam_destroy_region(descLocal[i]));
     }
+    EXPECT_NO_THROW(my_fam->fam_barrier_all());
+    if (*myPE == 0)
```

Comment:
Even if memory server profiling is disabled, rpc->generate_profile() should not create any problems. Did we test it ? We may not want to run memory server profiling always with microbenchmark. Can we put it within some condition ?